code/headers/hardware_usart.hpp

```diff
@@ -1,17 +1,18 @@
 #pragma once
 
-#include "hwlib.hpp"
+#include <hwlib.hpp>
 #include <queue.hpp>
 #include <uart_ports.hpp>
+#include <usart_connection.hpp>
```

Comment:
why are you including a file that is in the local directory as a library?

Reply:
What do you mean? according to the style guide we need to iclude with <> not ""

Reply:
in the build environment we include everything with <> in theory both will compile perfectly fine because the compiler findes the file both ways